packages/reports/tests/integration/components/unauthorized-table-test.js

```diff
@@ -29,6 +29,9 @@ module('Integration | Component | unauthorized table', function(hooks) {
 
     assert
       .dom('.navi-report-invalid__unauthorized')
-      .includesText('Protected Table', "Displays table name they don't have access to");
+      .includesText(
+        'You do not have access to run queries against . Please choose another table and run again.',
```

Comment:
Looks like this one isn't working as expected
```suggestion
        'You do not have access to run queries against Protected Table. Please choose another table and run again.',
```

Reply:
Ok I'll flip it back to failing